benchmark/README.rst

```diff
@@ -35,16 +35,17 @@ Comparing Revisions
 -------------------
 
 An additional script ``benchmark_revisions.py`` enables the comparative benchmarking of different
-revisions of PennyLane. It calls upon ``benchmark.py`` and thus supports all its different arguments.
+git revisions of PennyLane. It calls upon ``benchmark.py`` and thus supports all its different arguments.
 The revisions -- including branches, tags and commits -- are specified via ``-r revision1[,revision2[,revision3...]]``,
 as in
 
 .. code-block:: bash
 
   python3 benchmark_revisions.py -r master,0c8e90a -d default.qubit,default.tensor time bm_mutable_rotations
 
-The chosen revisions will be downloaded and cached into the ``revisions`` subdirectory of the benchmarking folder.
+The chosen revisions will be downloaded and cached into the ``pennylane.benchmark/revisions`` subdirectory of your user home directory.
```

Comment:
:+1:

Reply:
looks like it's `.pennylane/benchmarks/revisions`?